### DISCUSSION TITLE
compilation

### DISCUSSION THREAD

**Discussion by fredpacc** [strandborg/AlsavoCtrl#5] [General]
Hello, i see your work and it's very good.

i want to compiled your code but i have many errors.
i novice and what command do you use to compiled the code correctly?

Thanks 
regards
Fred

**Comment by strandborg** [comment]
The project uses CMake as the build system, so go to cmake.org and install the latest version. Once cmake.exe is available on your command prompt, run the following commands when in the AlsavoCtrl root directory:

```
mkdir build
cd build
cmake ..
cmake --build .
```
